specifications/base.protocol.md

```diff
@@ -0,0 +1,106 @@
+# Base Concepts
```

Comment:
```suggestion
# Base Concepts {#identity-protocol-base}
```
This avoids links from VPP and CIP breaking

Reply:
Fixed.